templates/etc/init.d/deb/td-agent

```diff
@@ -81,11 +81,8 @@ if [ -n "${TD_AGENT_PID_FILE}" ]; then
   TD_AGENT_ARGS="${TD_AGENT_ARGS} --daemon ${TD_AGENT_PID_FILE}"
 fi
 
-# 2012/04/17 Kazuki Ohta <k@treasure-data.com>
-# Use jemalloc to avoid memory fragmentation
-if [ -f "${TD_AGENT_HOME}/embedded/lib/libjemalloc.so" ]; then
-  export LD_PRELOAD="${TD_AGENT_HOME}/embedded/lib/libjemalloc.so"
-fi
+# Reduce memory usage. This gives better results than jemalloc with Ruby 2.6.
```

Comment:
Nit: this comment only makes full sense in the context of the diff (when you see that it used to use jemalloc). I'd suggest expanding on this (e.g., "than the previously used jemalloc" or something), or removing any mention of jemalloc and explaining what `MALLOC_ARENA_MAX` actually does.

Reply:
Done.